search/domain/base.py

```diff
@@ -201,8 +201,9 @@ class Document:
     def __post_init__(self) -> None:
         """Set latest_version, if not already set."""
         if not self.latest_version and self.latest:
-            if 'v' in self.latest:
-                self.latest_version = int(self.latest.split('v')[-1])
+            m = re.match(r'^(.+?)(?:v(?P<version>[\d]+))?$', self.latest)
+            if m.group('version'):
```

Comment:
[``re.match``](https://docs.python.org/3/library/re.html#re.match) may return ``None``, in which case this will raise an ``AttributeError``. Consider: ``if m and m.group('version'):`` ?

Reply:
Will do, thanks